### PR TITLE
[Lecture Topic Task]: Centralize Input Validation Utility Functions

### PR DESCRIPTION
**Pull request by jankii03**
Completed centralization for input validation utility functions.

### CODE REVIEW

**Review by devlin-hahn04 — APPROVED** [review]
As team lead, I approve of this pr

File: src/tests/utils/cases/TC-UTIL-VALIDATION-01_validation_utility_functions.ts
Comment (kian-robert) [inline src/tests/utils/cases/TC-UTIL-VALIDATION-01_validation_utility_functions.ts:1]: Please create the "cases" folder within this directory and add this document in there.

File: src/tests/utils/reports/TR-UTIL-VALIDATION-01_validation_utility_functions.adoc
Comment (kian-robert) [inline src/tests/utils/reports/TR-UTIL-VALIDATION-01_validation_utility_functions.adoc:1]: Please create the "reports" folder within this directory and add this document to it.

File: src/tests/utils/scripts/validation.test.ts
Comment (kian-robert) [inline src/tests/utils/scripts/validation.test.ts:1]: Please create the "scripts" folder within this directory and add this document to it.

File: src/tests/utils/TR-UTIL-VALIDATION-01_validation_utility_functions.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+. `formatPhoneNumber` returns an empty string for empty, whitespace-only, non-string, or digit-less inputs.
+
+== Notes
+Methodology: Equivalence Partitioning + Boundary Value Analysis.
+
+Run locally with:
+
+[source,bash]
+----
+npx jest src/tests/utils/validation.test.ts
```
Comment (kian-robert) [inline src/tests/utils/TR-UTIL-VALIDATION-01_validation_utility_functions.adoc]: Also quick detail, I am un sure if this command works without issue for you but for me I had the little issue that it got confused on the jest config file. To fix this I made it look directly into the correct directory so it did not get confused. If for some reason this issue suddenly appears please run the following command: `npx jest src/tests/utils/validation.test.ts --config jest.config.js` . If this command works fine for you as it did for me I suggest you change this line to the command I just provided. All tests ran smooth and all succeeded so once these minimal changes are done I can approve this Pr.

**Review by kian-robert — CHANGES_REQUESTED** [review]
Please review Comments for me to approve

**Review by kian-robert — APPROVED** [review]
As reviewer I approve this PR and it is ready to be merged.

**Review by TaimaraColon — APPROVED** [review]
This PR meets acceptance criteria and is ready to be merged. The associated issue will be closed upon merging.
